Formula/b/bitcoin.rb

```diff
@@ -1,8 +1,8 @@
 class Bitcoin < Formula
   desc "Decentralized, peer to peer payment network"
   homepage "https://bitcoincore.org/"
-  url "https://bitcoincore.org/bin/bitcoin-core-26.1/bitcoin-26.1.tar.gz"
-  sha256 "9164ee5d717b4a20cb09f0496544d9d32f365734814fe399f5cdb4552a9b35ee"
+  url "https://bitcoincore.org/bin/bitcoin-core-27.0/bitcoin-27.0.tar.gz"
+  sha256 "9c1ee651d3b157baccc3388be28b8cf3bfcefcd2493b943725ad6040ca6b146b"
   license "MIT"
   head "https://github.com/bitcoin/bitcoin.git", branch: "master"
 
```

Comment:
The two patches being applied should be removed now.

Reply:
👍